sqls/test6.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+    column_1,
+    column_2,
+    column_3,
+       column_4
```

Comment:
⚠️ **[sqlfluff]** <[LT02](https://docs.sqlfluff.com/en/stable/rules.html#sqlfluff.core.rules.Rule_LT02)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indent of 4 spaces.